### PR TITLE
Make the build faster for the CI

### PR DESCRIPTION
**Pull request by easternbloc**
There's a bug in kb8or which means it takes 10 mins to bring down each RC when you do a deploy.

This means the build will always be a min of 30mins (booo hiss, i hate my life etc etc) so this change makes dev and preprod both have 1 RC (meaning 10min builds)

This bug should be fixed with kb8or

### CODE REVIEW

File: kube/services/environments/preprod.yaml [hunk 1]
```diff
@@ -29,3 +29,4 @@ Fn::FileData:
 
 GRO_HTTP_PORT: 30243
 GRO_HTTPS_PORT: 30244
+REPLICAS: 1
```
Comment (daniel-ac-martin) [inline kube/services/environments/preprod.yaml]: I'd have 3 in preprod so you are replicating prod.

Comment (easternbloc) [inline kube/services/environments/preprod.yaml]: I knew you'd say this. But think of the build. My 😢 roll down my face every time the build runs 💧 💧 💧 💦

Comment (JoeChapman) [inline kube/services/environments/preprod.yaml]: 😿 💧

**Comment by JoeChapman** [comment]
🌵

**Comment by joefitter** [comment]
🚒

**Comment by daniel-ac-martin** [comment]
@JoeChapman: :camel:

**Comment by easternbloc** [comment]
SPEED!

**Pull request merged**